### PR TITLE
Issue #11777: Add prepareJSPThreadCount to the metatype.

### LINKED ISSUE #11777: prepareJSPThreadCount is not documented in Open Liberty - Investigate if any issues using it and document
https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/JspOptions.java#L92

If no issue in Open Liberty then we should add it to the metaType. First we need to test and validate the code path.

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #11777

### CODE REVIEW

**Comment by pnicolucci** [comment]
#run-libby-bot

**Comment by pnicolucci** [comment]
#run-libby-bot

**Comment by pnicolucci** [comment]
#build

File: dev/com.ibm.ws.jsp/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -48,4 +48,7 @@ scratchdir.name=Directory where generated class files are created
 scratchdir=When this attribute is set, the JSPs are compiled to the specified directory instead of the server workarea directory.
 
 useJDKCompiler.name=Compile JSPs with the Java SDK compiler
-useJDKCompiler=When this attribute is set, the JSPs are compiled by using the Java SDK compiler.
\ No newline at end of file
+useJDKCompiler=When this attribute is set, the JSPs are compiled by using the Java SDK compiler.
+
+prepareJSPThreadCount.name=The number of threads to use when translating and compiling JSPs on startup.
```
Comment (ManasiGandhi) [inline dev/com.ibm.ws.jsp/resources/OSGI-INF/l10n/metatype.properties:53]: Change,
"The number of threads to use when translating and compiling JSPs on startup."
to,
"The number of threads to use when you translate and compile JSPs on startup."

File: dev/com.ibm.ws.jsp/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -48,4 +48,7 @@ scratchdir.name=Directory where generated class files are created
...
+prepareJSPThreadCount=When prepareJSPs is enabled this configuration specifies the number of threads used.
```
Comment (ManasiGandhi) [inline dev/com.ibm.ws.jsp/resources/OSGI-INF/l10n/metatype.properties:54]: Change,
"When prepareJSPs is enabled this configuration specifies the number of threads used."
to,
"When prepareJSPs is enabled, this configuration specifies the number of threads used."

**Review by ManasiGandhi — CHANGES_REQUESTED** [review]
Few edit suggestions.

**Comment by pnicolucci** [comment]
Thanks @ManasiGandhi I've made those changes.

**Comment by pnicolucci** [comment]
Looks like the build failed @ManasiGandhi  so I need to change a couple of things with the labels as they are too long. Let me push up some edits and see if you're ok with it.

**Comment by pnicolucci** [comment]
#build

**Comment by skasund** [comment]
L2 message review not required

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
@ManasiGandhi  any comments on this? I had to rework a bit due to the length limitations.

**Comment by pnicolucci** [comment]
Talked with Manasi and she told me she was ok to merge so doing so.

**Review by wtlucy — APPROVED** [review]

**Comment by pnicolucci** [comment]
#run-libby-bot

**Pull request merged**